custom_components/automower/const.py

```diff
@@ -14,69 +14,140 @@
-        "icon": DEFAULT_ICON,
-        "message": "Cutting (manual timer override)",
-    },
-    STATUS_OK_LEAVING: {"icon": DEFAULT_ICON, "message": "Leaving charging station"},
-    STATUS_PAUSED: {"icon": "mdi:pause", "message": "Paused"},
-    STATUS_PARKED_TIMER: {"icon": DEFAULT_ICON, "message": "Parked due to timer"},
-    STATUS_PARKED_AUTOTIMER: {
-        "icon": DEFAULT_ICON,
-        "message": "Parked due to weather timer",
-    },
-    STATUS_PARKED_PARKED_SELECTED: {"icon": "mdi:sleep", "message": "Parked manually"},
-    STATUS_OK_SEARCHING: {
-        "icon": "mdi:magnify",
-        "message": "Going to charging station",
-    },
-    STATUS_EXECUTING_START: {"icon": "mdi:dots-horizontal", "message": "Starting..."},
-    STATUS_EXECUTING_STOP: {"icon": "mdi:dots-horizontal", "message": "Stopping..."},
-    STATUS_EXECUTING_PARK: {
-        "icon": "mdi:dots-horizontal",
-        "message": "Preparing to park...",
-    },
-    STATUS_OFF_HATCH_OPEN: {"icon": "mdi:alert", "message": "Hatch opened"},
-    STATUS_OFF_HATCH_CLOSED: {
-        "icon": "mdi:pause",
-        "message": "Stopped but not on base",
-    },
-    STATUS_OFF_DISABLED: {"icon": "mdi:close-circle-outline", "message": "Off"},
+    STATUS_ERROR:                   { 'icon': 'mdi:alert',          'message': 'Error' },
+    STATUS_OK_CHARGING:             { 'icon': 'mdi:battery-charging',     'message': 'Charging' },
+    STATUS_OK_CUTTING:              { 'icon': DEFAULT_ICON,         'message': 'Mowing' },
+    STATUS_OK_CUTTING_MANUAL:       { 'icon': DEFAULT_ICON,         'message': 'Mowing (manual timer override)' },
+    STATUS_OK_LEAVING:              { 'icon': DEFAULT_ICON,         'message': 'Leaving charging station' },
+    STATUS_PAUSED:                  { 'icon': 'mdi:pause',          'message': 'Paused' },
+    STATUS_PARKED_TIMER:            { 'icon': 'mdi:timetable',      'message': 'Parked (week timer)' },
+    STATUS_PARKED_AUTOTIMER:        { 'icon': 'mdi:weather-partly-cloudy',      'message': 'Parked (weather timer)' },
+    STATUS_PARKED_PARKED_SELECTED:  { 'icon': 'mdi:garage',          'message': 'Parked (manual)' },
+    STATUS_OK_SEARCHING:            { 'icon': 'mdi:magnify',        'message': 'Going to charging station' },
+    STATUS_EXECUTING_START:         { 'icon': 'mdi:dots-horizontal','message': 'Starting...' },
+    STATUS_EXECUTING_STOP:          { 'icon': 'mdi:dots-horizontal','message': 'Stopping...' },
+    STATUS_WAIT_POWER_UP:           { 'icon': 'mdi:dots-horizontal','message': 'Powering up...' },
+    STATUS_EXECUTING_PARK:          { 'icon': 'mdi:dots-horizontal','message': 'Preparing to park...' },
+    STATUS_OFF_HATCH_OPEN:          { 'icon': 'mdi:alert',          'message': 'Hatch opened' },
+    STATUS_OFF_HATCH_CLOSED:        { 'icon': 'mdi:pause',          'message': 'Stopped but not on base' },
+    STATUS_OFF_DISABLED:            { 'icon': 'mdi:close-circle-outline', 'message': 'Off'}
```

Comment:
The code should be formatted with black, in the same way as the whole HA code-base. I agree that your suggestion is more legible, but black wins.